cms/static/sass/views/_container.scss

```diff
@@ -277,8 +278,8 @@
           }
         }
 
-        .wrapper-tag-header:focus {
-          border: 1px dotted gray;
+        .wrapper-tag-header:focus-visible {
+          border-color: gray;
```

Comment:
Is there an SCSS variable we can use like `$gray` or `$border-color`? Just wondering.

Reply:
Updated https://github.com/openedx/edx-platform/pull/33884/commits/9f103537c583dc6924f8c9178a6b5f626cded672